src/runtime/composables/useAlgoliaSearch.ts

```diff
@@ -39,6 +39,10 @@ export function useAlgoliaSearch (indexName?: string) {
     return searchResult
   }
 
+  onUnmounted(() => {
+    result.value = undefined;
+  });
```

Comment:
Hey, what about instead to provide a `reset` function so that the user can trigger this reset function when he needs? :)

It can even be the same (on `onUnmounted` hook) but it will be done by the consumer of the library instead of the library itself :)

Reply:
Actually that makes more sense! :)

Reply:
Great, could you refactor your code to this change? Then I could merge, test and release a new version with it to unblock you :)

Reply:
@Baroshem Can you continue here maybe? I dont have the time unfortunately